docs/gen-docs/kyma_alpha_enable_module.md

```diff
@@ -0,0 +1,56 @@
+---
+title: kyma alpha enable module
+---
+
+Enables a module in the cluster or in the given Kyma resource.
+
+## Synopsis
+
+Use this command to enable Kyma modules available in the cluster.
+
+### Detailed description
+
+For more information on Kyma modules, see the 'create module' command.
+
+This command enables an available module in the cluster. 
+A module is available when it is released with a ModuleTemplate. The ModuleTemplate is used for instantiating the module with proper default configuration.
+
+
+```bash
+kyma alpha enable module [name] [flags]
+```
+
+## Examples
+
+```bash
+
+Enable "my-module" from "alpha" channel in "default-kyma" in "kyma-system" Namespace
+		kyma alpha enable module my-module -c alpha -n kyma-system -k default-kyma
+
+```
+
+## Flags
+
+```bash
+  -c, --channel string     The name of the module's channel to enable.
+  -f, --force-conflicts    Force the patching of Kyma spec modules in case its managed field has already been edited by another source then Kyma CLI.
```

Comment:
```suggestion
  -f, --force-conflicts    Forces the patching of Kyma spec modules in case their managed field was edited by a source other than Kyma CLI.
```